setup.py

```diff
@@ -10,11 +9,9 @@
     # py_modules=['hello'],
     # scripts=['bin/nlp-evaluate'],
 
-    packages=[
-        'nlpsandboxclient'
-    ],
+    packages=setuptools.find_packages(),
```

Comment:
https://stackoverflow.com/questions/50521626/how-to-use-find-packages-to-package-all-files-from-subdirectory